From charrer@alacritech.com  Tue Mar 17 15:56:07 2009
From: Mithlesh Thukral <mithlesh@linsyssoft.com>
Date: Mon, 16 Mar 2009 08:34:14 -0400
Subject: Staging: sxg: Fix the module in Kconfig file for Sahara SXG driver
To: "'Greg KH'" <gregkh@suse.de>, "'Greg KH'" <greg@kroah.com>
Cc: "'Mithlesh Thukral'" <mithlesh@linsyssoft.com>
Message-ID: <D2656CDCFC8745DB9B87E6B3D51AF846@exton.alacritech.com>


From: Mithlesh Thukral <mithlesh@linsyssoft.com>

* Update the module name in Kconfig help section.
  This is a classic case of documentation keeping out of pace with
  development
  And this was overlooked by me ages ago when we had fixed the Makefile for
  sxg_ethtool to compile.

Signed-off-by: Mithlesh Thukral <mithlesh@linsyssoft.com>
Cc: Christopher Harrer <charrer@alacritech.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/sxg/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/sxg/Kconfig
+++ b/drivers/staging/sxg/Kconfig
@@ -8,4 +8,4 @@ config SXG
 	  10Gbe network cards.
 
 	  To compile this driver as a module, choose
-	  M here: the module will be called sxg.
+	  M here: the module will be called sxg_nic.
